lib/constants.ts

```diff
@@ -42,14 +51,6 @@ export const GAME_CATEGORIES = {
 
 export const BASE_TITLE = 'Warzone Stats Tracker';
 export const SENDER_EMAIL = 'warzone@brshooter.com'
-export const GAMER_TABLE = 'gamers';
```

Comment:
@fjaxyu all Tables and views should be referred to as `VIEWS.` or `TABLES.` instead of doing constants like this.

Reply:
I think this was legacy stuff, before I made the TABLES and VIEWS constants